install_dependencies.sh

```diff
@@ -32,7 +32,28 @@ echo "Installing Misc. Utilities"
 echo "================================================================"
 
 sudo apt-get install -y\
-    clang-format
+    clang-format\
+    python-rosinstall
+
+
+echo "================================================================"
+echo "Installing Project Dependent ROS packages."
+echo "================================================================"
+
+# Setup rosinstall
+mkdir -p extended_pkg
+rosinstall extended_pkg /opt/ros/kinetic .rosinstall
+rosinstall .
```

Comment:
Could we add more detailed comments for each of these calls to `rosinstall`? Just to make it a bit more clear for someone who's not super familiar with it.

Reply:
done